rmf_robot_sim_gz_plugins/src/TeleportDispenser.cpp

```diff
@@ -249,26 +455,24 @@ void TeleportDispenserPlugin::Configure(const Entity& entity,
     rclcpp::init(0, argv);
 
   _dispenser = entity;
-  _dispenser_common->guid =
-    ecm.Component<components::Name>(_dispenser)->Data();
-  gzwarn << "Initializing plugin with name " << _dispenser_common->guid <<
-    std::endl;
+  _guid = ecm.Component<components::Name>(_dispenser)->Data();
+  gzwarn << "Initializing plugin with name " << _guid << std::endl;
```

Comment:
A bit leftover from a long time ago but can we downgrade this to an info message rather than a warning?